config_jira.go

```diff
@@ -38,17 +39,22 @@ func (c *JiraConfig) Get(ctx context.Context) error {
 func (c *JiraConfig) Set(ctx context.Context) error {
 	return errors.Wrapf(setConfigSection(ctx, c.SectionId(), bson.M{
 		"$set": bson.M{
-			"host":       c.Host,
-			"basic_auth": c.BasicAuthConfig,
-			"oauth1":     c.OAuth1Config,
-			"email":      c.Email,
+			"host":                  c.Host,
+			"basic_auth":            c.BasicAuthConfig,
+			"oauth1":                c.OAuth1Config,
+			"personal_access_token": c.PersonalAccessToken,
+			"email":                 c.Email,
 		}}), "updating config section '%s'", c.SectionId(),
 	)
 }
 
 func (c *JiraConfig) ValidateAndDefault() error {
-	if (c.Host != "") && (c.BasicAuthConfig.Username != "") == (c.OAuth1Config.AccessToken != "") {
-		return errors.New("must specify exactly 1 Jira auth method")
+	hostPopulated := c.Host != ""
+	basicAuthPopulated := c.BasicAuthConfig.Username != ""
+	oauth1Populated := c.OAuth1Config.AccessToken != ""
+	patPopulated := c.PersonalAccessToken != ""
+	if hostPopulated && !basicAuthPopulated && !oauth1Populated && !patPopulated {
```

Comment:
!hostPopulated I believe

Reply:
This one actually has to be `hostPopulated` - some tests initialize the global admin settings without setting explicit Jira credentials (only required for Jira-specific integration tests), so checking if the host is populated is a proxy for determining if the Jira config should be used. The pre-change logic also had this check, but with a slightly more confusing boolean statement.